### ISSUE TITLE
Call Script --help in Travis Test

### ISSUE THREAD

**Issue by ZLLentz** [pcdshub/pcds-envs#96]
At minimum, we need to call the `--help` function on each script we expect from our home-grown packages to catch obvious errors. `pytmc --help` in the most recent environment outputs:
```
Traceback (most recent call last):
  File "/reg/neh/home/zlentz/miniconda3/envs/pcds-2.3.4/bin/pytmc", line 6, in <module>
    from pkg_resources import load_entry_point
  File "/reg/neh/home/zlentz/miniconda3/envs/pcds-2.3.4/lib/python3.6/site-packages/pkg_resources/__init__.py", line 3251, in <module>
    @_call_aside
  File "/reg/neh/home/zlentz/miniconda3/envs/pcds-2.3.4/lib/python3.6/site-packages/pkg_resources/__init__.py", line 3235, in _call_aside
    f(*args, **kwargs)
  File "/reg/neh/home/zlentz/miniconda3/envs/pcds-2.3.4/lib/python3.6/site-packages/pkg_resources/__init__.py", line 3264, in _initialize_master_working_set
    working_set = WorkingSet._build_master()
  File "/reg/neh/home/zlentz/miniconda3/envs/pcds-2.3.4/lib/python3.6/site-packages/pkg_resources/__init__.py", line 583, in _build_master
    ws.require(__requires__)
  File "/reg/neh/home/zlentz/miniconda3/envs/pcds-2.3.4/lib/python3.6/site-packages/pkg_resources/__init__.py", line 900, in require
    needed = self.resolve(parse_requirements(requirements))
  File "/reg/neh/home/zlentz/miniconda3/envs/pcds-2.3.4/lib/python3.6/site-packages/pkg_resources/__init__.py", line 786, in resolve
    raise DistributionNotFound(req, requirers)
pkg_resources.DistributionNotFound: The 'epics-pypdb' distribution was not found and is required by pytmc
```
And catching this on travis would have allowed me to avoid hotfixing a deployed environment.

**Comment by klauer** [comment]
@hhslepicka and I were talking about getting a PCDS-maintained fork of pypdb with the name `epics-pydb` (as it's on PyPI) and also a conda package of our own.

Upstream does not appear to be interested in packaging outside of debian.

**Comment by hhslepicka** [comment]
Upstream is interested in renaming and even adopting a tag naming that is in line with Python package. We are discussing this but let's proceed with the fork and later on we can send changes upstream.

**Comment by klauer** [comment]
TLDR: @hhslepicka will take care of the details 😁

**Comment by ZLLentz** [comment]
I'm not worried about this particular instance. Here I'm just noting that it's a good idea to call your scripts before saying that the environment is OK to deploy, using pytmc as an example because I was using it to understand what the hole in my integration testing was.
